cdmtaskservice/job_state.py

```diff
@@ -108,19 +108,17 @@ async def submit(self, job_input: models.JobInput, user: CTSUser) -> str:
                 f"Job compute time of {compute_time} CPU hours is greater than the limit of "
                 + f"{self._cpu_hrs}"
         )
-        if not self._test_mode:
-            # check the flow is available before we make any changes
-            flow = await self._flowman.get_flow(job_input.cluster)
-            # TODO SECURITY this needs to be moved to job flow providers when they exist, so
-            #               it can be checked if the job flow doesn't exist
```

Comment:
Not sure what I was thinking here but
* the difference would be between a message saying the flow isn't available vs. the user isn't allowed to use the flow, so doesn't seem like a big deal to me
* now the method is a preflight method anyway, and may make changes to the db